devel/ccache-devel/Portfile

```diff
@@ -25,24 +26,23 @@ long_description \
     compilation can be satisfied from cache.  This often results in a 5 to 10 \
     times speedup in common compilations.
 
-homepage            https://ccache.samba.org/
+homepage            https://ccache.dev
+
+# gcc 4.x clang < 500 does not understand -Wpedantic
+compiler.blacklist *gcc-4.* {clang < 500}
```

Comment:
Instead of restricting the compiler so much, could `-Wpedantic` be removed?

Reply:
Well yes, I guess you could, but personally, I always favour leaving the compiler options in that upstream wishes to use, and pick a compiler that supports them.

Reply:
Well I prefer to allow older compilers if they work. `-Wpedantic` is a flag that enables certain warnings. Warnings are intended for developers. Developers are welcome to enable all the warnings they want on their own systems, but MacPorts is for users who just want to install software with a minimum of fuss, and for me that includes not burdening the user with extra dependencies (like compilers) that aren't really needed.

Reply:
Well, I guess its your port ;) I don't really agree though, I fail to see the advantage of patching things to use what is now an anyway an archaic compiler. Just forcing a more modern compiler seems better, to me. I'll look into it though if you insist.

Reply:
Done.